server/resp/src/main/java/org/infinispan/server/resp/Resp3Handler.java

```diff
@@ -29,10 +31,18 @@ public class Resp3Handler extends Resp3AuthHandler {
    private static final Log log = LogFactory.getLog(MethodHandles.lookup().lookupClass(), Log.class);
    static final byte[] OK = "+OK\r\n".getBytes(StandardCharsets.US_ASCII);
 
+   protected AdvancedCache<byte[], byte[]> ignoreLoaderCache;
+
    Resp3Handler(RespServer respServer) {
       super(respServer);
    }
 
+   @Override
+   protected void setCache(AdvancedCache<byte[], byte[]> cache) {
+      super.setCache(cache);
+      ignoreLoaderCache = cache.withFlags(Flag.SKIP_CACHE_LOAD);
```

Comment:
`IGNORE_RETURN_VALUE` can be set as well.

Reply:
True, I can add this in another PR.